e2e/terraform/volumes.tf

```diff
@@ -1,5 +1,5 @@
 resource "aws_efs_file_system" "csi" {
-  creation_token = "CSI"
+  creation_token = "${local.random_name}-CSI"
```

Comment:
Terraform seems to generate a unique value already by default:

> `creation_token` - (Optional) A unique name (a maximum of 64 characters are allowed) used as reference when creating the Elastic File System to ensure idempotent file system creation. By default generated by Terraform. See Elastic File System user guide for more information.
> https://www.terraform.io/docs/providers/aws/r/efs_file_system.html#creation_token

Should we rely on terraform generated values instead of setting a value here?

Reply:
It's kind of a bizarre API requirement from AWS given they normally rely on request IDs generated by the SDKs for idempotency, isn't it? But yeah if Terraform does the right thing by default, we can probably just drop this line.